lib/src/rocket.rs

```diff
@@ -531,7 +548,11 @@ impl Rocket {
     /// ```
     #[inline]
     pub fn catch(mut self, catchers: Vec<Catcher>) -> Self {
-        info!("👾  {}:", Paint::purple("Catchers"));
+        if self.config.use_emoji {
+            info!("👾  {}:", Paint::purple("Catchers"));
```

Comment:
This alien monster should use `Paint::masked`; I believe it not being masked before was just an oversight.